src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$day-size: 100px;
+$day-color: #eee;
+$gap-size: 1px;
+$columns: 7;
+$hovered-day-color: #ffc0cb;
+$mindays: 28;
+$maxdays: 31;
```

Comment:
The variable name $maxdays should be in camelCase or with hyphens, like $maxDays or $max-days for better readability.